css/default.css

```diff
@@ -1065,853 +1065,852 @@ div.language-plaintext,
 }
 
 .language-bash:before {
-    content: "Bash";
+  content: "Bash";
 }
 
 .documenttitle {
-    display: flex;
+  display: flex;
 }
 
 .documenttitle .h1 {
-    align-self: center;
+  align-self: center;
 }
 
 .pagehr {
-    border: 0;
-    border-top: 1px solid #ddd;
-    margin-top: 20px;
+  border: 0;
+  border-top: 1px solid #ddd;
+  margin-top: 20px;
 }
 
 .sidebarhr {
-    margin-left: 20.5px;
-    margin-right: 20.5px;
-    height: 2px;
-    background-color: #616d8a;
-    border-top: 0;
+  margin-left: 20.5px;
+  margin-right: 20.5px;
+  height: 2px;
+  background-color: #616d8a;
+  border-top: 0;
 }
 
 .m-t-2 {
-    margin-top: 28px;
+  margin-top: 28px;
 }
 
 div#defaultsidebar .mysidebar .sidebarbuttoncontainer {
-    display: flex;
-    flex-direction: column;
-    padding-left: 0;
+  display: flex;
+  flex-direction: column;
+  padding-left: 0;
 }
 
 #mysidebar ul.sidebarbuttoncontainer a.bottombuttons.spacebetweenHyperlink {
-    padding: 0;
-    margin: 0 !important;
+  padding: 0;
+  margin: 0 !important;
 }
 
 .supportbutton {
-    padding: 10px;
-    margin-left: 39px;
-    font-family: Inter, sans-serif;
-    font-size: 14px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    justify-self: end;
-    margin-right: 39px;
-    border: solid 1px #f2f1f2;
-    text-align: center;
+  padding: 10px;
+  margin-left: 39px;
+  font-family: Inter, sans-serif;
+  font-size: 14px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  justify-self: end;
+  margin-right: 39px;
+  border: solid 1px #f2f1f2;
+  text-align: center;
 }
 
 .supportbutton span {
-    font-family: Inter, sans-serif;
-    font-size: 14px;
-    color: #f2f1f2;
+  font-family: Inter, sans-serif;
+  font-size: 14px;
+  color: #f2f1f2;
 }
 
 .defaultbutton {
-    font-family: Inter, sans-serif;
-    font-size: 14px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    justify-self: end;
-    margin-left: 39px;
-    margin-right: 39px;
-    text-align: center;
-    margin-top: 20px;
-    background-color: #a9acb4;
+  font-family: Inter, sans-serif;
+  font-size: 14px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  justify-self: end;
+  margin-left: 39px;
+  margin-right: 39px;
+  text-align: center;
+  margin-top: 20px;
+  background-color: #a9acb4;
 }
 
 .defaultbutton:hover {
-    background-color: #162036;
+  background-color: #162036;
 }
 
 .defaultbutton a {
-    color: var(--link-color);
+  color: var(--link-color);
 }
 
 .defaultbutton div {
-    padding: 10px;
-    color: var(--darker-gray);
-    width: 100%;
+  padding: 10px;
+  color: var(--darker-gray);
+  width: 100%;
 }
 
 .defaultbutton div:hover {
-    color: #fff;
+  color: #fff;
 }
 
 #jumpto {
-    flex: 1;
+  flex: 1;
 }
 
 #anchorlist {
-    margin-top: 90px;
-    text-align: left;
-    position: fixed;
-    max-height: 360px;
-    overflow-wrap: anywhere;
-    overflow-y: scroll;
```

Comment:
https://developers.liveperson.com/agent-workspace-widget-sdk-public-model-structure.html

this is needed, look at this page, i think the right code will be something like this, will need to look, but if you remove the scroll, once you get to the bottom, there will be hidden links

#anchorlist {
    margin-top: 90px;
    text-align: left;
    position: fixed;
    height: 100%;
    overflow-wrap: anywhere;
    overflow-y: auto;
}